code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -498,28 +498,6 @@
 	REMOVE_TRAIT(L, TRAIT_STUNRESISTANCE, type)
 	..()
 
-/datum/reagent/medicine/mental_stabilizator
-	name = "Mental Stabilizator"
-	description = "Heals any potential issues with mental state of the patient."
-	reagent_state = LIQUID
-	color = "#CCFFFF"
-	metabolization_rate = 0.5 * REAGENTS_METABOLISM
-	overdose_threshold = 25
-
-/datum/reagent/medicine/mental_stabilizator/on_mob_life(mob/living/M)
-	if(!ishuman(M))
-		return
-	var/mob/living/carbon/human/H = M
-	H.adjustSanityLoss(5*REM) // That's healing
-	..()
-	. = 1
-
-/datum/reagent/medicine/sal_acid/overdose_process(mob/living/M)
```

Comment:
This seems to have been removed and not replaced anywhere?

Reply:
Never mind, it seems that was a duplicate that was left over... which would explain why Mental Stabilizers literally DIDN'T have an Overdose